web/src/main/java/org/openmrs/web/Listener.java

```diff
@@ -424,10 +429,31 @@ private void copyCustomizationIntoWebapp(ServletContext servletContext, Properti
 					}
 				}
 			}
-			
+			List<String> locales = Arrays.asList("", "_de", "_en_GB", "_es", "_fa", "_fr", "_hi", "_in_ID", "_it", "_kh",
```

Comment:
At a very quick glance, this doesn't include the ht or rw locales, both of which are important to PIH.

Personally I don't think that being able to edit message bundles on the fly without restarting the application is important, but apparently others have thought that, because we've spent work on it.

Using the allowed locales makes sense.